test/squish/suite_VerifySettingsDialog/shared/scripts/gui_helpers.py

```diff
@@ -0,0 +1,166 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# -*- coding: utf-8 -*
+# mypy: disable-error-code="attr-defined"
+
+import gui_locators
+import squish
+import test
+
+snoozeTimeout = 3  # seconds
+
+
+def launchDeadlineConfigGUI():
+    squish.startApplication("deadline config gui")
+    test.log("Launching Deadline Config GUI")
+    test.log(
+        "Sleep for " + str(snoozeTimeout) + " seconds to allow GUI authentication to fully load."
+    )
+    squish.snooze(snoozeTimeout)
+    test.compare(
+        squish.waitForObjectExists(
+            gui_locators.aWS_Deadline_Cloud_workstation_configuration_DeadlineConfigDialog
+        ).visible,
+        True,
+        "Expect the Deadline Config GUI to be open.",
+    )
+
+
+def setGlobalSettings(profileName: str, jobHistDir: str):
+    test.compare(
+        str(squish.waitForObjectExists(gui_locators.profile_settings_QLineEdit).displayText),
+        jobHistDir,
+        "Expect job history directory default path to be equal.",
+    )
+    squish.mouseClick(
+        squish.waitForObjectItem(gui_locators.global_settings_AWS_profile_QComboBox, profileName),
+        484,
+        9,
+        squish.Qt.NoModifier,
+        squish.Qt.LeftButton,
+    )
+    squish.mouseClick(
+        squish.waitForObjectItem(gui_locators.global_settings_AWS_profile_QComboBox, profileName),
+        225,
+        6,
+        squish.Qt.NoModifier,
+        squish.Qt.LeftButton,
+    )
+    test.log("Set Global settings")
+
+
+def setProfileSettings(farmName: str):
```

Comment:
Is there a test here that we can successfully open the files browser?

Reply:
Will add in the next revision. Good catch. Thanks!